src/status_im/ui/screens/about_app/views.cljs

```diff
@@ -3,19 +3,52 @@
   (:require [status-im.ui.components.toolbar.view :as toolbar]
             [status-im.ui.components.react :as react]
             [status-im.ui.components.status-bar.view :as status-bar]
+            [status-im.ui.screens.about-app.styles :as styles]
             [status-im.i18n :as i18n]
+            [status-im.transport.utils :as transport.utils]
             [status-im.ui.screens.profile.components.views :as profile.components]
             [re-frame.core :as re-frame]))
 
+(defn app-info->json [app-version peers mailserver node-info node-version]
+  (js/JSON.stringify
+   (clj->js
+    {:client {:app-version app-version
+              :mailserver mailserver
+              :node-info node-info
+              :node-version node-version}
+     :peers peers})
+   nil
+   4))
+
+(defn peer-view [{:keys [enode]}]
+  (let [[enode-id ip-address port] (transport.utils/extract-url-components enode)]
+    ^{:key enode}
+    [react/touchable-highlight {:style styles/peer-view
+                                :on-press #(react/copy-to-clipboard enode)}
+     [react/view
+      [react/view
```

Comment:
are the nested view necessary?

Reply:
I think so:

"TouchableHighlight must have one child (not zero or more than one). If you wish to have several child components, wrap them in a View" 

Otherwise they all display inline.

Reply:
ok I was talking about the text items I think you are right it would display inline